ui/src/main/java/org/jtaccuino/core/ui/JavaCellFactory.java

```diff
@@ -317,7 +321,9 @@ private void handleExecution() {
                                         var result = new Label(resultData);
                                         result.getStyleClass().add("jshell_eval_result");
                                         outputBox.getChildren().add(result);
-                                        control.getCellData().getOutputData().add(new CellData.OutputData(org.jtaccuino.core.ui.api.CellData.OutputData.OutputType.DISPLAY_DATA, Map.of("text/plain", resultData)));
+                                        control.getCellData().getOutputData().add(
+                                                CellData.OutputData.of(org.jtaccuino.core.ui.api.CellData.OutputData.OutputType.DISPLAY_DATA,
```

Comment:
Couldn't you use the enum directly as the import for the enclosing class is there?
```suggestion
                                                CellData.OutputData.of(CellData.OutputData.OutputType.DISPLAY_DATA,
```

Reply:
Needs rework with all those enclosing classes. Something for a tidy up things PR.